opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -45,4 +45,18 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successfull export request.
+  // When the request is only partially successfull (due to e.g. invalid/bad data),
+  // the server MUST populate the details property with a developer-facing error
+  // message along with the number of accepted log records.
+  ExportLogsPartialSuccess details = 1;
```

Comment:
This will be null when it is used with older proto versions, right? :thinking:

Reply:
You mean when parsing a response from a server that used an older proto version? If so yes, this will be null, thus the need to check as in the example.